projects/plugins/jetpack/extensions/blocks/blogroll/blogroll.php

```diff
@@ -76,9 +76,32 @@ function site_recommendations_settings() {
 		'general',
 		'Blogroll Recommendations', // Visible to the user see: https://github.com/WordPress/gutenberg/issues/41637
 		array(
-			'type'          => 'array',
-			'show_in_rest'  => true,
 			'description'   => __( 'Site Recommendations', 'jetpack' ),
+			'type'          => 'array',
+			'show_in_rest'  => array(
+				'schema' => array(
+					'items' => array(
+						'type'       => 'object',
+						'properties' => array(
+							'id'          => array(
+								'type' => 'string',
+							),
+							'name'        => array(
+								'type' => 'string',
+							),
+							'icon'        => array(
+								'type' => 'string',
+							),
+							'url'         => array(
+								'type' => 'string',
```

Comment:
```suggestion
								'type'   => 'string',
								'format' => 'uri',
```
The REST API supports a `format` arg here, that allows devs to be [more specific about sanitization requirements](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api.php#L2827-L2852) for certain strings.

For any values from user input you can use `'text-field'` for it to be run through `sanitize_text_field()`.